src/style.css

```diff
@@ -1,3 +1,202 @@
+@import url('https://fonts.googleapis.com/css2?family=Abril+Fatface&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
-  background-color: bisque;
+  /* background: url(./assets/background.jpg); */
```

Comment:
- I will like to kindly suggest that you remove unnecessary comments from your code base to make more readable😀